### PR TITLE
test: update profiles for DirectPath & traffic director

### PR DESCRIPTION
**Pull request by igorbernstein2**
Adapt DirectPath integration tests to Traffic Director:
* DirectPath ConnectionModes now enable attemptXds
* Add a new connection mode for CloudPath over Traffic Director

### CODE REVIEW

File: google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java [hunk 1]
```diff
@@ -239,6 +241,7 @@ private void configureUserAgent(EnhancedBigtableStubSettings.Builder stubSetting
         // nothing special
         break;
       case REQUIRE_CFE:
```
Comment (kolea2) [inline google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java:243]: to confirm, is this expected to change the settings for the `REQUIRE_CFE` case?

**Review by kolea2 — COMMENTED** [review]
lgtm after one confirmation

File: google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java (hunk 1 above)
Comment (igorbernstein2) [reply google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java:243]: yep

**Comment by igorbernstein2** [comment]
closing in favor of #2263

**Pull request closed without merging**